lab-ronoh/route/note-router.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const Router = require('express').Router;
+const debug = require('debug')('note:note-router');
+const bodyParser = require('body-parser').json();
+
+// const AppError = require('../lib/app-error');
+const storage = require('../lib/storage');
+const Note = require('../model/note');
+const sendError = require('../lib/error-response');
+
+const noteRouter = module.exports = new Router();
+
+function createNote(reqBody){
+  debug('createNote');
+  return new Promise(function(resolve, reject){
+    var note;
+    try{
+      note = new Note(reqBody.content);
+    }catch(err){
+      reject(err);
+    }
+    storage.setItem('note', note).then(function(note){
```

Comment:
It might be more readable if you formatter your .thens and .catches to be at the same level of indentation.